cmd/ethrex/cli.rs

```diff
@@ -355,6 +355,14 @@ pub async fn import_blocks(
     genesis: Genesis,
     evm: EvmEngine,
 ) -> Result<(), ChainError> {
+    // If the genesis.json file is pre-Paris return a custom error
```

Comment:
we don't want this validation only when importing blocks, we want it in general. So I think we should do this validation when initializing a `Genesis`.